app/src/main/res/values/styles.xml

```diff
@@ -429,12 +426,9 @@
   </style>
   <!-- Continue Lesson Button -->
   <style name="ContinueLessonButton" parent="TextAppearance.AppCompat.Widget.Button">
-    <item name="android:layout_width">144dp</item>
+    <item name="android:layout_width">0dp</item>
```

Comment:
Would it be preferrable, in this case(refer to above comment on constraint layout), to have these attributes set at view level rather than at style level?

Reply:
I can certainly try to do that, but I would like to know what is the reason why the former code author put those attributes here instead of the view level. And to clarify, if it is just layout width or height or all the items in the style? I also see there are other style settings for edittext etc. are those also gonna be a problem?

Reply:
The attributes were placed in syles for reusability. However, since the style naming is pretty specific, it seems this style is specific to this button, and in this case, it is okay to move the width and height attributes to the view level. This will make it easier to debug should the parent layout change.